### DISCUSSION TITLE
What type of face swap does fooocus use? I want to replicate it on usual stable diffusion (in case if stable diffusion doesn't support controlnet)

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#3480] [Q&A] [answered]
What type of face swap does fooocus use? I want to replicate it on usual stable diffusion (in case if stable diffusion doesn't support controlnet)

**Comment by Yogesh-DevHub** [comment, marked as answer]
Hi, 
Actually, it resembles the image we want to be in result using face swap. 
A short explanation has been given in the video https://www.youtube.com/watch?v=fnhjSC4T69o

However, mixing image with inpaint, I feel does a better job of face swap 
A 2min example given here: https://youtu.be/lkZbyGTwxkM?si=7o4iok5BIYrTPVqV
